CONTRIBUTING.md

```diff
@@ -11,3 +11,33 @@ Read and follow the coding style and guidelines [from this file](CODING_STYLE_GU
 - Don't forget to update any impacted CHANGELOG file(s)
 - Run the *fix_files.sh* script
 - Start a github pull request
+
+## To run `fix_files.sh` on MAC
+
+* Update `bash` from 3.x to latest (5.x)
+
+The following commands will alter default system shell for the current user to freshly installed bash.
+
+```bash
+brew install bash
+echo "/usr/local/bin/bash" | sudo tee -a /etc/shells
+chsh -s /usr/local/bin/bash
+```
+
+You can revert to any shell, listed by 
+
+```bash
+cat /etc/shells
+```
+
+including default system `/bin/bash` or `/bin/zsh` with the following command
+
+```bash
+chsh -s <your preferred shell>
+```
+
+* Install latest `clang-format` (11.x)
+
+```bash
+brew install romansavrulin/clang-format/clang-format-lambda --HEAD
```

Comment:
So, I found a few minutes to try out your version (I mean, try cf-11.0) and it suffer from the same bug than original 7.0 version from which I derived my patch from.
Although it seems that this repo only have one function call that is problematic, I'm using the same cf file and version everywhere (more than 50 projects), shared with dozens of developers. Other projects are heavily using parameters before lambda, and are exposed to this formatting issue.

I ran my test cases (the one I linked in the previous PR) with your version and compared to mine, and it's indeed faulty:
```
$ udiff a.txt b.txt
--- a.txt       2020-02-26 15:13:01.000000000 +0100
+++ b.txt       2020-02-26 15:14:03.000000000 +0100
@@ -14,13 +14,15 @@
                call();
        },
        5);
-paramBeforeAndAfterLambda(8,
+paramBeforeAndAfterLambda(
+       8,
        [](int x)
        {
                call();
        },
        5);
-doubleLambda(8,
+doubleLambda(
+       8,
        [](int x)
        {
                call();

```

What I suggest is take as much as possible from your fixes now, then I'll have to find the time to grab the latest cf source code, make a new patch over it to fix this remaining issue, finally take your changes to .clang-format
Does it sound ok?

Reply:
I guess we should file this test case to the [original contributor's patch](https://reviews.llvm.org/D44609).

Reply:
or make a PR directly to  [llvm repo](https://github.com/llvm/llvm-project)

Reply:
Well, it did already more than 1.5y ago :)
The issue is that the bug is not in his patch actually, but in clang-format base code. I hoped the OP would have forwarded the issue.

Reply:
> or make a PR directly to [llvm repo](https://github.com/llvm/llvm-project)

Yeah I unfortunately don't have the time to look deeply into clang-format. Like I said in my answers in the llvm post, I made a quick&dirty fix that seemed to work without side effects on all my projects, but I'm not sure it's the proper way to fix the issue as I don't know the code architecture of the project.
I won't be able to find the time to make a proper PR that will be accepted by llvm.
The only thing I can do is file a bug and hope someone will do it properly (I already provided test cases and patch fix).

I'll see what I can do about it though, just not immediately

Reply:
> Well, it did already more than 1.5y ago :)
yep, but it has settled just 12 days ago. So OP could be still encouraged enough to fix this minor too.

Reply:
I know :)
I was on vacations when I happilly received emails on the conversation, telling the fix was accepted.
I just posted something in the post, we'll see how it goes.

For now, I'll have to wait before changing clang-format version on **all** my projects.
Let's keep this PR open so I can come back to it when I'll either find the time to do something about it, or it's finally fixed in llvm.
I will need to take the latest version of llvm soon anyway so it might help me push this forward.

Reply:
That's ok to wait for now. But if we have to wait too long, I'd prefer to take this steps:

- I can make home-brew recipe for the clang-format version you use, if you provide me the exact repos and commits over which your patch could be applied.
- Or I can make home-brew recipe for the fresh clang-format version with new patches to fix your formatting issue.
- We can put this homebrew tap into l-acoustics organization on GitHub  to let users quickly bootstrap their dev env.

Reply:
brew tap for our organization is something I was definitely considering and will probably happen regardless of the rest :)

IIRC the exact version and patch can be found in the root of this repository (patch is committed and version is provided in one of the .md file)